docs/how-tos/hello-world-module.md

```diff
@@ -0,0 +1,508 @@
+
+   For name, you can use the automatic `camera-1`.
+
+1. Configure the image path attribute by pasting the following in place of the `{}` brackets:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+    "image_path": "<replace with the path to your image>"
+   }
+   ```
+
+   Replace the path with the path to your image, for example `"/Users/jessamyt/Downloads/hello-world.jpg"`.
+
+1. Save the config, then click the **TEST** section of the camera's configuration card.
+
+   ![The Viam app configuration interface with the Test section of the camera card open, showing a hello world image.](/how-tos/hello-camera.png)
+
+   You should see your image displayed.
+   If not, check the **LOGS** tab for errors.
+
+{{< expand "Click if you also created a sensor component" >}}
+
+9. Add the modular sensor:
+
+   Click **+**, click **Local module**, then click **Local component**.
+
+   For the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet" >}}, enter `<namespace>:hello-world:hello-sensor`, replacing `<namespace>` with the organization namespace you used when generating the stub files.
+   For example, `jessamy:hello-world:hello-sensor`.
+
+   For type, enter `sensor`.
+
+   For name, you can use the automatic `sensor-1`.
+
+10. Save the config, then click **TEST** to see a random number generated every second.
+
+    ![The sensor card test section open.](/how-tos/hello-sensor.png)
+
+{{< /expand >}}
+
+## Package and upload the module
+
+You have a working local module.
+You can make it available to deploy on more machines by packaging it and uploading it to the [Viam registry](https://app.viam.com/registry).
+
+The hello world module you created is for learning purposes, not to provide any meaningful utility, so we recommend making it available only to machines within your {{< glossary_tooltip term_id="organization" text="organization" >}} instead of making it publicly available.
+
+{{< expand "Click to see what you would do differently if this wasn't just a hello world module" >}}
+
+1. Create a GitHub repo with all the source code for your module.
+   Add the link to that repo as the `url` in the <file>meta.json</file> file.
+1. Create a README to document what your module does and how to configure it.
+1. If you wanted to share the module outside of your organization, you'd set `"visibility": "public"` in the <file>meta.json</file> file.
+
+{{< /expand >}}
+
+To package and upload your module and make it available to configure on machines in your organization:
+
+1. Package the module as an archive, run the following command from inside the <file>hello-world</file> directory:
+
+    ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+    tar -czf module.tar.gz run.sh setup.sh requirements.txt src
+    ```
+
+    This creates a tarball called <file>module.tar.gz</file>.
+
+1. Run the `viam module upload` CLI command to upload the module to the Viam registry:
```

Comment:
```suggestion
1. Run the `viam module upload` CLI command to upload the module to the registry:
```
it's either the registry or the Viam Registry. If you're refering to it as the product